.github/workflows/int-and-connected-test-run.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Run Connected test via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.39.0
+        uses: broadinstitute/datarepo-actions@rc-se-actions-testing
```

Comment:
Should we wait until the action is merged?

Reply:
Yes! I'll change this before merging in the PR (whenever we agree to do the upgrade)